Production_scripts/Create_slf_lookups/lookup_gp_cluster.R

```diff
@@ -63,9 +61,9 @@ gp_clusters_clean <- matched_gp_clusters %>%
 
 gp_clusters_clean %>%
   # .zsav
-  write_sav(get_slf_gp_cluster_path(update = latest_update, ext = "zsav", check_mode = "write")) %>%
+  write_sav(get_slf_gp_cluster_path(ext = "zsav", check_mode = "write")) %>%
   # .rds file
```

Comment:
It looks good. Just a writing permission issue but this issue seems quite personal.

`Error: [EPERM] Failed to chmod '/conf/hscdiip/SLF_Extracts/Lookups/practice_details_Mar_2022.zsav': operation not permitted `

Reply:
Not sure why you are getting this error, as the permissions look ok `-rw-rw---- ` and since I'm the owner the `write_rds` or `write_sav` function shouldn't have tried to change the permissions...

I've actually removed the changes to this script as I realised it was a duplicate and so will just remove it on a separate branch.

Reply:
The error is because the file was created before the writing permission issue was fixed by James. Jennifer and I have tested it and it's fine now.